src/services/item/plugins/h5p/test/plugin.test.ts

```diff
@@ -32,7 +33,7 @@ async function cleanFiles() {
 
 describe('Service plugin', () => {
   let app: FastifyInstance;
-  let member: any;
+  let member: Member | null;
```

Comment:
Use `Actor`?

Reply:
Not equivalent since `Actor = Member | undefined` but `null` is not treated as the same type